__brick__/.eslintrc.json

```diff
@@ -7,6 +7,11 @@
     "simple-import-sort"
   ],
   "rules": {
+    "@typescript-eslint/strict-boolean-expressions": "off",
+    "@typescript-eslint/no-misused-promises": "off",
```

Comment:
Non consumare una promise non è una best practice.. Perché stiamo spegnendo questa rule? Qual è il caso d'uso in cui ci rema contro?

https://typescript-eslint.io/rules/no-misused-promises/